packages/nextjs/src/config/index.ts

```diff
@@ -35,7 +35,11 @@ class Config {
   }
 
   get baseUrl(): string {
-    return getEnv(EnvVariables.FRONTEGG_BASE_URL) ?? setupEnvVariables.FRONTEGG_BASE_URL;
+    const baseUrl = getEnv(EnvVariables.FRONTEGG_BASE_URL) ?? setupEnvVariables.FRONTEGG_BASE_URL;
+    if (baseUrl.endsWith('/')) {
```

Comment:
isn't is affect any other customer that is waiting for the slash? in any other usage?

Reply:
If you provide the baseUrl with '/' the app should crash, therefore it shouldn't break old behaior